client/src/app/app.module.ts

```diff
@@ -26,6 +26,8 @@ import {DashboardComponent} from './components/dashboard/dashboard.component';
 import {SettingNewPasswordComponent} from './components/setting-new-password/setting-new-password.component';
 import {RequestPasswordResetComponent} from './components/request-password-reset/request-password-reset.component';
 import {TemplateComponent} from './components/template/template.component';
+import {ConditionComponent} from './components/condition/condition.component';
```

Comment:
Could you standardise all these imports for consistency - when Adam did his he went for { x } so probably best for this to be consistent across the board 👍